x/accounts/keeper.go

```diff
@@ -86,7 +86,7 @@ func NewKeeper(
 		AccountNumber:   collections.NewSequence(sb, AccountNumberKey, "account_number"),
 		AccountsByType:  collections.NewMap(sb, AccountTypeKeyPrefix, "accounts_by_type", collections.BytesKey, collections.StringValue),
 		AccountByNumber: collections.NewMap(sb, AccountByNumber, "account_by_number", collections.BytesKey, collections.Uint64Value),
-		AccountsState:   collections.NewMap(sb, implementation.AccountStatePrefix, "accounts_state", collections.BytesKey, collections.BytesValue),
+		AccountsState:   collections.NewMap(sb, implementation.AccountStatePrefix, "accounts_state", collections.PairKeyCodec(collections.Uint64Key, collections.BytesKey), collections.BytesValue),
```

Comment:
It'd be good to add a godoc or inline comment mentioning the keys and value

Reply:
this good: https://github.com/cosmos/cosmos-sdk/pull/18664/files#diff-c86def99d2ff8d9493d9ce55f86fbdba6b07ece4d3569b33d4911db0104d6274R130 ?

I can add more docs in case it's not good, lmk